advanced_source/cpp_custom_ops.rst

```diff
@@ -255,13 +244,41 @@ first load the C++ library that holds the custom operator definition
 and then call the ``torch.library`` registration APIs. This can happen in one
 of two ways:
 
-1. If you're following this tutorial, importing the Python C extension module
-   we created will load the C++ custom operator definitions.
-2. If your C++ custom operator is located in a shared library object, you can
-   also use ``torch.ops.load_library("/path/to/library.so")`` to load it. This
-   is the blessed path for Python agnosticism, as you will not have a Python C
-   extension module to import. See `torchao __init__.py <https://github.com/pytorch/ao/blob/881e84b4398eddcea6fee4d911fc329a38b5cd69/torchao/__init__.py#L26-L28>`_
-   for an example.
+
+1. In this tutorial, our C++ custom operator is located in a shared library object,
+   and we use ``torch.ops.load_library("/path/to/library.so")`` to load it. This
+   is the blessed path for Python agnosticism, and you will not have a Python C
+   extension module to import. See our `extension_cpp/__init__.py <https://github.com/pytorch/extension-cpp/blob/e4c4eb822889ea67f191071fa627d750e04bf047/extension_cpp/__init__.py>`_
```

Comment:
I'm curious: can one still import a module with py_limited_api=True?

Reply:
yes lol

it worked fine with ao, but no promises on other variations of PYBIND11_MODULE

Reply:
Defining [Py_LIMITED_API](https://docs.python.org/3/c-api/stable.html#c.Py_LIMITED_API) in the header should be sufficient to prevent the code from compiling if someone uses a non-py-limited-api.

So, I think we don't need to recommend torch.ops.load_library. Thoughts?

In general in open source I have preferred asking users to import c-extension modules over calling torch.ops.load_library, because there are various footguns associated with it

Reply:
Also, if the pybind11_module is broken with Py_LIMITED_API, then there's also a standard cpython way of creating the extension module

Reply:
fixed it!